sos/report/plugins/libraries.py

```diff
@@ -44,7 +44,9 @@ def setup(self):
                 dirs.add(fqlib[1].rsplit('/', 1)[0])
 
             if dirs:
-                self.add_cmd_output(f"ls -lanH {' '.join(dirs)}",
-                                    suggest_filename="ld_so_cache")
+                self.add_dir_listing(
+                    f"{' '.join(dirs)}",
+                    suggest_filename='ld_so_cache'
+                )
```

Comment:
We will stop printing UIDs/GIDs here (dropping `-n` option), no idea if / how much it can matter. I expect no big deal - can @rmetrich as author of https://github.com/sosreport/sos/commit/ff74942afb5b1f0d2b840204325f709fa0896452 confirm?